.github/ISSUE_TEMPLATE/1-bug.yaml

```
@@ -0,0 +1,39 @@
+name: Bug report
+description: Message parser doesn't parse right
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for helping to improve DeltaChat! Please fill information below to make it easier to kill this bug!
```

I still think people don't get the joke about "killing" bugs.

```suggestion
      value: Thanks for helping to improve DeltaChat! Please fill information below to make it easier to us to understand and solve the issue.
```